src/test/java/th/ac/kmitl/science/comsci/example/models/CompanyTest.java

```diff
@@ -66,6 +66,7 @@ public void getAllDataFromAddressInstanceCase(){
         address.setCitySubDivision("Makkasan");
         address.setCity("Ratchatewee");
         address.setCountrySubDivision("Bangkok");
+        address.setCountry("Thailand");
         
         assert (address.getLine1().contains("1103/52"));
         assert (address.getLine2().contains("Pratunam Clinic"));
```

Comment:
Why we use contains for all tests here ? Is it mutate the property setting via Address class ?
If not, why we not use equal ?